README.md

```diff
@@ -77,6 +52,14 @@ make deploy
      - **MERKLE_ROOT_ROLE**: Updates the Merkle root.
      - **WALLET_ROLE**: Sets the rewards wallet.
 
+## Merkle Tree Assumptions
+For the spark-rewards cumulative claims to function properly, the Merkle tree is expected to adhere to the following properties:
+
+1. The Merkle tree can contain several epochs.
+2. The Merkle tree's leaves are not removed unless the leaves' epoch is permanently closed.
```

Comment:
We don't need to remove anything. This data structure is meant to store rediculous amounts of data.